docs/tutorials/get-started/single-component-tutorials-servo-mousemover.md

```diff
@@ -0,0 +1,274 @@
+---
+title: "Single Component Tutorial: How to Create a Mouse Mover Using a Servo"
+linkTitle: "Create a Mouse Mover"
+weight: 40
+type: "docs"
+description: "Use the Python SDK to move a continuous servo to keep a computer screen from sleeping."
+webmSrc: "/tutorials/img/single-component-tutorials-servo-mousemover/pi-hole.webm"
+mp4Src: "/tutorials/img/single-component-tutorials-servo-mousemover/pi-hole.mp4"
+videoAlt: "A mouse mover controlled with a servo and Raspberry Pi."
+tags: ["servo", "single component tutorial", "raspberry pi", "sdk", "python"]
+# SME: Kacey Meier-Smith
+---
+
+Have you or someone you've known been a person that works from home and needed to step out for a moment?
+Maybe it was to clear your head, grab your lunch, answer the door, and you get in trouble because it showed you as offline?
+
+<div class="embed-responsive embed-responsive-16by9">
+<iframe width="984" height="554" src="https://www.youtube.com/embed/sXWJE0ePHdA" title="Mouse Mover Teaser" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+</div>
+
+This tutorial will show you how to build a mouse mover using Viam, a Raspberry Pi, a servo, and an optical mouse.
+This robot will turn the continuous servo that's secured inside the box, which will turn the circle under the optical mouse. This will keep your computer from falling asleep.
+
+This project is a good place to begin if you're new to robots and would like to learn how to use a [servo component](/components/servo/) with the Viam app and Viam's [Python SDK](https://python.viam.dev/).
+
+<div class="td-max-width-on-larger-screens">
+  {{<gif webm_src="../../img/single-component-tutorials-servo-mousemover/moving.webm" mp4_src="../../img/single-component-tutorials-servo-mousemover/moving.mp4" alt="Video of a mouse running erratically on the screen in a sweeping motion in front of the Viam app Control page with an overlaid video of a mouse on top of a cardboard box with a moving cardboard circle underneath it with red swirl lines on the circle.">}}
+</div>
+
+{{< alert title="Tip" color="tip" >}}
+NOTE: This tutorial is part of Viam's Single Component Tutorial series, where you have the opportunity to learn how to use single components for low cost with Viam's platform in fun mini-tutorials.
+
+Stay tuned for more tutorials like this one and don't forget to [join our Discord](https://discord.gg/viam) if you have any questions in the meantime!
+{{< /alert >}}
```

Comment:
Please remove this for now. This isn't a series yet - and it's confusing for people to see this touted as a series. I understand it is meant to become one. Once it is you can add this here but for now it shouldn't be here. 
```suggestion
```